amlb/results.py

```diff
@@ -308,6 +313,16 @@ def save_predictions(dataset: Dataset, output_file: str,
 
         df = df.assign(predictions=preds)
         df = df.assign(truth=truth)
+        if quantiles is not None:
```

Comment:
Should this be inside the `if dataset.type == DatasetType.timeseries:` block?

Reply:
Should we raise exception if quantiles != None and problem type is not timeseries?